backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/carousel/BpkCarousel.kt

```diff
@@ -30,12 +30,27 @@ import androidx.compose.ui.platform.testTag
 import net.skyscanner.backpack.compose.pageindicator.BpkPageIndicator
 import net.skyscanner.backpack.compose.pageindicator.BpkPageIndicatorStyle
 
-@OptIn(ExperimentalFoundationApi::class)
 @Composable
 fun BpkCarousel(
     state: BpkCarouselState,
     modifier: Modifier = Modifier,
     content: @Composable (BoxScope.(Int) -> Unit),
+) {
+    BpkCarousel(
+        state = state,
+        modifier = modifier,
+        content = content,
+        overlayContent = { pageIndicator -> Box(Modifier.align(Alignment.BottomCenter)) { pageIndicator?.invoke() } },
+    )
+}
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+internal fun BpkCarousel(
```

Comment:
created an internal reusable function - the gallery carousel has different spacing + more content for the overlay